common/log/tagValues.go

```diff
@@ -0,0 +1,132 @@
+	TagValueActionCompleteWorkflow                tagValueTypeWorkflowAction = "add-complete-workflow-event"
+	TagValueActionFailWorkflow                    tagValueTypeWorkflowAction = "add-fail-workflow-event"
+	TagValueActionTimeoutWorkflow                 tagValueTypeWorkflowAction = "add-timeout-workflow-event"
+	TagValueActionCancelWorkflow                  tagValueTypeWorkflowAction = "add-cancel-workflow-event"
+	TagValueActionTimerStarted                    tagValueTypeWorkflowAction = "add-timer-started-event"
+	TagValueActionTimerFired                      tagValueTypeWorkflowAction = "add-timer-fired-event"
+	TagValueActionTimerCanceled                   tagValueTypeWorkflowAction = "add-timer-Canceled-event"
+	TagValueActionWorkflowTerminated              tagValueTypeWorkflowAction = "add-workflowexecution-terminated-event"
+	TagValueActionWorkflowSignaled                tagValueTypeWorkflowAction = "add-workflowexecution-signaled-event"
+	TagValueActionContinueAsNew                   tagValueTypeWorkflowAction = "add-continue-as-new-event"
+	TagValueActionWorkflowCanceled                tagValueTypeWorkflowAction = "add-workflowexecution-canceled-event"
+	TagValueActionChildExecutionStarted           tagValueTypeWorkflowAction = "add-childexecution-started-event"
+	TagValueActionStartChildExecutionFailed       tagValueTypeWorkflowAction = "add-start-childexecution-failed-event"
+	TagValueActionChildExecutionCompleted         tagValueTypeWorkflowAction = "add-childexecution-completed-event"
+	TagValueActionChildExecutionFailed            tagValueTypeWorkflowAction = "add-childexecution-failed-event"
+	TagValueActionChildExecutionCanceled          tagValueTypeWorkflowAction = "add-childexecution-canceled-event"
+	TagValueActionChildExecutionTerminated        tagValueTypeWorkflowAction = "add-childexecution-terminated-event"
+	TagValueActionChildExecutionTimedOut          tagValueTypeWorkflowAction = "add-childexecution-timedout-event"
+	TagValueActionRequestCancelWorkflow           tagValueTypeWorkflowAction = "add-request-cancel-workflow-event"
+	TagValueActionWorkflowCancelRequested         tagValueTypeWorkflowAction = "add-workflow-execution-cancel-requested-event"
+	TagValueActionWorkflowCancelFailed            tagValueTypeWorkflowAction = "add-workflow-execution-cancel-failed-event"
+	TagValueActionWorkflowSignalRequested         tagValueTypeWorkflowAction = "add-workflow-execution-signal-requested-event"
+	TagValueActionWorkflowSignalFailed            tagValueTypeWorkflowAction = "add-workflow-execution-signal-failed-event"
+	TagValueActionUnknownEvent                    tagValueTypeWorkflowAction = "add-unknown-event"
+)
+
+// Pre-defined values for TagWorkflowListFilterType
+const (
+	TagValueListWorkflowFilterByID     tagValueTypeWorkflowListFilterType = "WID"
+	TagValueListWorkflowFilterByType   tagValueTypeWorkflowListFilterType = "WType"
```

Comment:
The difference in casing looks odd to me it feels like it should be (wtype and status) or (WType and Status). This is just opinion though.

Reply:
Good catches. I just copy it from existing code without noticing that. I will change it as you say